frontend/src/i18n/translation.json

```diff
@@ -8682,10 +8682,58 @@
+        "fr": "Vous pouvez personnaliser OpenHands pour votre dépôt en utilisant un micro-agent. Demandez à OpenHands de mettre une description du dépôt, y compris comment exécuter le code, dans .openhands/microagents/repo.md.",
+        "tr": "Bir mikro ajan kullanarak OpenHands'i deponuz için özelleştirebilirsiniz. OpenHands'ten deponun açıklamasını, kodun nasıl çalıştırılacağı dahil, .openhands/microagents/repo.md dosyasına koymasını isteyin.",
+        "uk": "Ви можете налаштувати OpenHands для свого репозиторію за допомогою мікроагента. Попросіть OpenHands розмістити опис репозиторію, включаючи інформацію про те, як запустити код, у файлі .openhands/microagents/repo.md."
+    },
+    "CONVERSATION$SHOW_MICROAGENTS": {
+        "en": "Show Microagents",
+        "ja": "マイクロエージェントを表示",
+        "zh-CN": "显示微代理",
+        "zh-TW": "顯示微代理",
+        "ko-KR": "마이크로에이전트 표시",
+        "no": "Vis mikroagenter",
+        "ar": "عرض الوكلاء المصغرين",
+        "de": "Mikroagenten anzeigen",
+        "fr": "Afficher les micro-agents",
+        "it": "Mostra microagenti",
+        "pt": "Mostrar microagentes",
+        "es": "Mostrar microagentes",
+        "tr": "Mikro ajanları göster",
+        "uk": "Показати мікроагентів"
+    },
+    "CONVERSATION$NO_MICROAGENTS": {
+        "en": "No microagents found for this conversation.",
+        "ja": "この会話用のマイクロエージェントが見つかりませんでした。",
+        "zh-CN": "未找到此对话的微代理。",
+        "zh-TW": "未找到此對話的微代理。",
+        "ko-KR": "이 대화에 대한 마이크로에이전트를 찾을 수 없습니다.",
+        "no": "Ingen mikroagenter funnet for denne samtalen.",
+        "ar": "لم يتم العثور على وكلاء مصغرين لهذه المحادثة.",
+        "de": "Keine Mikroagenten für dieses Gespräch gefunden.",
+        "fr": "Aucun micro-agent trouvé pour cette conversation.",
+        "it": "Nessun microagente trovato per questa conversazione.",
+        "pt": "Nenhum microagente encontrado para esta conversa.",
+        "es": "No se encontraron microagentes para esta conversación.",
+        "tr": "Bu konuşma için mikro ajan bulunamadı.",
+        "uk": "Для цієї розмови не знайдено мікроагентів."
+    },
+    "CONVERSATION$FAILED_TO_FETCH_MICROAGENTS": {
+        "en": "Failed to fetch microagents",
+        "ja": "マイクロエージェントの取得に失敗しました",
+        "zh-CN": "获取微代理失败",
+        "zh-TW": "獲取微代理失敗",
+        "ko-KR": "마이크로에이전트를 가져오지 못했습니다",
+        "no": "Kunne ikke hente mikroagenter",
+        "ar": "فشل في جلب الوكلاء المصغرين",
+        "de": "Fehler beim Abrufen von Mikroagenten",
+        "fr": "Échec de la récupération des micro-agents",
+        "it": "Impossibile recuperare i microagenti",
+        "pt": "Falha ao buscar microagentes",
+        "es": "Error al obtener microagentes",
+        "tr": "Mikro ajanlar getirilemedi",
+        "uk": "Не вдалося отримати мікроагентів"
```

Comment:
@OpenHands please use the wording "available microagents" instead of just "microagents"